pkg/pillar/cmd/upgradeconverter/upgradeconverter.go

```diff
@@ -0,0 +1,54 @@
+package upgradeconverter
+
+import (
+	"flag"
+	"log"
+)
+
+type ConverterAction uint8
+
+const (
+	Convert ConverterAction = iota + 1
+	CleanupOld
+	CleanupNew
+	NoAction
+)
+
+var conversionHandlers []ConversionHandler
+
+func Run() {
+
+	actionStr := flag.String("actionStr", "", "Choose an action from convert/cleanupOld/cleanupNew")
+	action := Convert
+	if *actionStr == "convert" {
+		action = Convert
+	} else if *actionStr == "cleanupOld" {
+		action = CleanupOld
+	} else if *actionStr == "cleanupNew" {
+		action = CleanupNew
+	} else {
+		action = NoAction
+	}
+	flag.Parse()
+
+	conversionHandlers = []ConversionHandler{
+		{
+			description: "Convert Global Settings to new format",
+			handlerFunc: handleUpgradeGlobalConfig,
+		},
+	}
```

Comment:
Declare this static array outside of "Run" function - it would be much more readable.